install-bundles/install-bundle-autopilot-namespaced/0-cnrm-system.yaml

```diff
@@ -1509,6 +1509,12 @@ rules:
   verbs:
   - create
   - patch
+- apiGroups:
```

Comment:
Did this change come from an [RBAC annotation](https://book.kubebuilder.io/reference/markers/rbac) ?  I think it should, but I don't see the change.

Reply:
This didn't come from RBAC annotation, will fix. Thanks for the suggestion.

Reply:
Removed udpates from generated YAML files and updated template "config/installbundle/releases/scopes/namespaced/shared-components/unmanageddetector_clusterrole.yaml"